server/paths/assess.ts

```diff
@@ -10,7 +12,7 @@ export default {
     index: assessApplicationsPath,
     show: singleApplicationPath,
     overview: singleApplicationPath.path('overview'),
-    addNote: singleApplicationPath.path('add-note'),
+    addNote: singleAssessmentPath.path('add-note'),
```

Comment:
hmm the ordering of these might be getting a bit confusing now, I wonder if it should be lifted out of `submittedApplications` and follow the pattern set for statusUpdates and assessmentDetails by giving notes their own top level key?
```
notes: {
  new: singleAssessmentPath.path('add-note')
}

```

Reply:
These changes no longer required since UI endpoints have been reverted back to as they were.  Only the API endpoint is required to be changed.